imports/api/classes/PushNotifManager.jsx

```diff
@@ -0,0 +1,71 @@
+import Util from '/imports/api/classes/Utilities';
+import { Meteor } from 'meteor/meteor';
+import moment from 'moment';
+import { PushNotifDB } from '../pushNotifications';
+
+export default class PushNotifManager {
+    constructor(body) {
+        this.json = {
+            server_key: body.server_key || null,
+            registration_id: body.registration_id || null,
+            title: body.title,
+            body: body.body,
+            icon: body.icon || '',
+            click_action: body.action || '',
+            priority: body.priority || 10,
+            createdTimestamp: moment().valueOf(),
+            price:0,
+            result:""
+        };
+        this.apiUrl = 'https://fcm.googleapis.com/fcm/';
```

Comment:
Please create a separate class for this, check old source code. PushNotifManager should only handle parsing, inserting and updating record in mongo db, nothing else